std/utf.d

```diff
@@ -1905,6 +1908,14 @@ unittest
 }
 /* =================== Encode ======================= */
 
+private dchar _utfException(UseReplacementDchar useReplacementDchar)(string msg, dchar c)
```

Comment:
I don't know why Go decided to invent "Rune" to be "CodePoint", but whatever. In any case, Go uses FFFD as well ('`RuneError`'), not some other character.